packages/near-membrane-base/src/environment.ts

```diff
@@ -219,15 +225,16 @@ export class VirtualEnvironment {
             blueCallableDebugInfo,
             noop, // blueCallableDefineProperties,
             blueCallableGetLazyPropertyDescriptorStateByTarget,
+            blueCallableGetPropertyValue,
             blueCallableGetTargetIntegrityTraits,
             blueCallableGetToStringTagOfTarget,
-            blueCallableGetTypedArrayIndexedValue,
```

Comment:
We now make this callable a bit more generically named so it's now `blueCallableGetPropertyValue` which does essentially `return target[key]`.